lib/src/main/java/io/ably/lib/transport/ConnectionManager.java

```diff
@@ -71,6 +71,11 @@ public class ConnectionManager implements ConnectListener {
     static ErrorInfo REASON_REFUSED = new ErrorInfo("Access refused", 401, 40100);
     static ErrorInfo REASON_TOO_BIG = new ErrorInfo("Connection closed; message too large", 400, 40000);
 
+    /**
+     * Flag that indicating that we entered termination state
+     */
+    private boolean terminating = false;
```

Comment:
```suggestion
    /**
     * Flag indicating existing active connection with the server
     */
    private boolean activeConnection = false;
```
`terminating` word does feel a bit misleading.

Reply:
@coderabbitai wdyt about this?

Reply:
wdyt about this @coderabbitai

Reply:
Well, this will ofc need inversion of `terminating` checks/assignment at other places as well.

Reply:
I will leave it to @ttypic for renaming the variable. There is ofc an option to rename it to `activeConnection=false`
or `active=false`